roadmap/implementers-guide/src/runtime/disputes.md

```diff
@@ -75,18 +74,18 @@ Frozen: bool,
   1. If there is no dispute under `Disputes`, create a new `DisputeState` with blank bitfields.
   1. If `concluded_at` is `Some`, and is `concluded_at + config.post_conclusion_acceptance_period < now`, return false.
   1. If the overlap of the validators in the `DisputeStatementSet` and those already present in the `DisputeState` is fewer in number than `byzantine_threshold + 1` and the candidate is not present in the `Included` map
```

Comment:
Does _overlap_ mean _intersection_ or _union_ here? Either way I don't think either works perfectly.

Assuming the first message is less than the threshold, in the intersection case, the second could be a disjoint set and hence be empty and marked as spam.

Assuming the union, and `q`-size small subsets (`total / q`) it's marking only the first `q-1` spammers as spammers.

Could elaborate a bit on the intended spam mechanics?

Reply:
That wording clarifies it 👍